### PR TITLE
initial app skeleton for the IdP

### PR DESCRIPTION
**Pull request by enriquepablo**
initial app skeleton for the IdP

### CODE REVIEW

File: src/eduid_webapp/idp/__init__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 NORDUnet A/S
```
Comment (johanlundberg) [inline src/eduid_webapp/idp/__init__.py]: Should probably update the copyright to "Copyright (c) 2019 SUNET".

File: src/eduid_webapp/idp/helpers.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+
+
+def is_healthy(self) -> dict:
```
Comment (johanlundberg) [inline src/eduid_webapp/idp/helpers.py]: The status check code resides in eduid_common.api.views.

File: src/eduid_webapp/idp/views.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import simplejson
+from flask import Blueprint, current_app
+
+from eduid_webapp.idp.helpers import is_healthy
+
+
+idp_views = Blueprint('idp', __name__, url_prefix='')
+
+
+@idp_views.route('/healthy', methods=['GET'])
```
Comment (johanlundberg) [inline src/eduid_webapp/idp/views.py]: The status check code resides in eduid_common.api.views.

File: src/eduid_webapp/idp/__init__.py (hunk 1 above)
Comment (enriquepablo) [reply src/eduid_webapp/idp/__init__.py]: fixed

File: src/eduid_webapp/idp/helpers.py (hunk 2 above)
Comment (enriquepablo) [reply src/eduid_webapp/idp/helpers.py]: Right you are, removed

File: src/eduid_webapp/idp/views.py (hunk 3 above)
Comment (enriquepablo) [reply src/eduid_webapp/idp/views.py]: removed

**Pull request merged**